documentation/4.0/content/introduction/platforms/environments.md

```diff
@@ -9,11 +9,12 @@ weight: 6
 
 ### Overview
 
-The operator supports a wide range of on-premises and cloud Kubernetes
-offerings where Kubernetes is supplied for you or you set up Kubernetes
-yourself. These include, but are not limited to:
+The operator supports running on production-quality Kubernetes environments, including on-premises,
```

Comment:
Yes this might be redundant with what we say in lines 32-33

Reply:
I want to distinguish between supported and certified and I think you are doing that in link 17 and 32.

Reply:
@mriccell ok, I'll consolidate them.